src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -126,6 +131,12 @@ public override async Task UpdatedAsync(UpdateContentContext context, AutorouteP
                     ContentItem = part.ContentItem
                 };
 
+                var cultureAspect = await _contentManager.PopulateAspectAsync(part.ContentItem, new CultureAspect());
+                if (cultureAspect.Culture != null)
```

Comment:
I don't think the `cultureAspect.Culture` will be `null` while it set to the `CurrentUICulture` in `CultureAspect`